test/platform.ts

```diff
@@ -62,6 +67,11 @@ export interface IEmulatorManager {
      * Prepares the emulator for a test.
      */
     prepareEmulatorForTest(appId: string): Q.Promise<string>;
```

Comment:
The platform interface would be a good place for the simulator/emulator related operations.